agent/agent.go

```diff
@@ -53,45 +53,61 @@ func main() {
 		os.Exit(1)
 	}
 
-	var previousCluster, previousInstanceID, previousContainerInstanceArn string
-	previousTaskEngine := engine.NewTaskEngine(cfg)
-	// previousState is used to verify that our current runtime configuration is
-	// compatible with our past configuration as reflected by our state-file
-	previousState, err := initializeStateManager(cfg, previousTaskEngine, &previousCluster, &previousContainerInstanceArn, &previousInstanceID)
-	if err != nil {
-		log.Crit("Error creating state manager", "err", err)
-		os.Exit(1)
-	}
+	var currentInstanceID, containerInstanceArn string
```

Comment:
should you call it currentEc2InstanceID and previousEc2InstanceID to be more explicit about which instance we're talking about?